lib/libvfio-user.c

```diff
@@ -476,90 +476,102 @@ consume_fd(int *fds, size_t nr_fds, size_t index)
 }
 
 int
-handle_dma_map_or_unmap(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
+handle_dma_map(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg,
+               struct vfio_user_dma_region *region)
 {
-    struct vfio_user_dma_region *dma_regions = msg->in_data;
-    bool is_map = (msg->hdr.cmd == VFIO_USER_DMA_MAP);
-    size_t nr_dma_regions;
-    size_t fdi;
-    size_t i;
+    char rstr[1024]; /* TODO this can be much smaller */;
```

Comment:
nit, please either dial down, or just remove the TODO, it's not really a big deal.

Reply:
Sure, I'll just remove the TODO.